sqlglot/optimizer/simplify.py

```diff
@@ -729,31 +729,43 @@ def _simplify_binary(expression, a, b):
     return None
 
 
-def simplify_parens(expression):
+def simplify_parens(expression: exp.Expression, dialect: DialectType) -> exp.Expression:
```

Comment:
I'd make this optional, so that it's backwards-compatible.
```suggestion
def simplify_parens(expression: exp.Expression, dialect: DialectType = None) -> exp.Expression:
```